howto/instance/create.md

```diff
@@ -71,6 +71,20 @@ Launch a raw instance by providing the image ID in the following command:
 
 See [Provided images](https://discourse.ubuntu.com/t/provided-images/24185) for a list of images that are available in Anbox Cloud.
 
+### Launch an instance with a specific name
+
+*since 1.22.0*
+
+To allow identifying and referencing instances easier you can give them a name at creation time. Names must be unique and can be used with all CLI commands.
```

Comment:
Could we add some more information on what characters can the name contain? Is it ASCII-only? UTF-8? Are some special characters disallowed? Is there a maximum length?
This may not fit the how-to section and may be better suited for the reference section, though.

Reply:
Agree on the additional information required about the instance name. However, if there is just a little more information about the naming conventions, it should reside here along with the relevant topic for context. I doubt if naming convention for an instance has content enough to warrant a separate reference topic at the moment. 

"Names must be unique and can be used with all CLI commands." - This can also be included in the list of conditions that instance names must fulfill.

Reply:
That is a good point but something we need to implement more generally IMHO as we have the same restriction applying for names of all objects. I can create a story to cover that.